docs/reference/modules/discovery/adding-removing-nodes.asciidoc

```diff
@@ -135,3 +135,28 @@ DELETE /_cluster/voting_config_exclusions
 DELETE /_cluster/voting_config_exclusions?wait_for_removal=false
 --------------------------------------------------
 // CONSOLE
+
+[[modules-discovery-removing-data-nodes]]
+==== Removing data nodes [WIP // DRAFT]
+When removing data nodes, it is important not to remove too many at the same time. The way of decommissioning a node is to instruct Elasticsearch to move shards away from that node. You just need to make sure that you have enough room in the cluster to accommodate the shards. Also remember to wait for  Elasticsearch to move shards around before shutting down the node.
+
+1. Validate node settings, including allocation <<allocation-awareness>> configurations
+2. Ensure the <<cluster-health>> is green
+3. Tell the cluster to move shards away (exclude) from the node being removed
++
+[source,js]
+--------------------------------------------------
+# Do not allocate shards by moving shards away (excluding) from the node
+# Define `cluster.routing.allocation.exclude.{attribute}` using one of the 
+# supported attributes:  `_name`, `_ip`, and `_host`
```

Comment:
In general, we avoid including comments in the example snippets in favor of incorporating the content into the intro or using callouts. It's not necessary to repeat what can easily be gleaned by reading the example itself.